src/components/ods-tables/table-component-tags/table-foot/ods-table-foot.scss

```diff
@@ -0,0 +1,3 @@
+:host {
```

Comment:
I've never see this CSS selector before. How does this work?

Reply:
instead of using a div in the render() of the tsx file we use a Host tag. Similar to <> or <Fragment>. But unlike Fragment( which I believe behaves unexpectedly) we can use the Host tag as an css selector. I forgot all about it until Daniel shared a stackoverflow post with me. That fixed the table display issues.